### PR TITLE
card production for g2HDM ttt analysis

### PR DESCRIPTION
**Pull request by efeyazgan**

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/g2HDM/ttt/g2HDM_ttt_a0_run_card.dat [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+#*********************************************************************
+# Matching parameter (MLM only)
+#*********************************************************************
+ 1 = ickkw            ! 0 no matching, 1 MLM
+ 1.0 = alpsfact         ! scale factor for QCD emission vx
+ False = chcluster        ! cluster only according to channel diag
+ 5 = asrwgtflavor     ! highest quark flavor for a_s reweight
+ True  = auto_ptj_mjj  ! Automatic setting of ptj and mjj if xqcut >0
+                                   ! (turn off for VBF and single top processes)
+ 30.0   = xqcut   ! minimum kt jet measure between partons
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/g2HDM/ttt/g2HDM_ttt_a0_run_card.dat:54]: is the xqcut already optimized with the qcut value

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/g2HDM/ttt/g2HDM_ttt_a0_proc_card.dat [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+#add process p p > t t  j / s0 QCD= 99 , (t > W+ b , W+ > l+ vl), (t > \
+#W+ b , W+ > l+ vl)
+#add process p p > t~ t~ / s0 QCD= 99 , (t~ > W- b~ , W- > l- vl~),   (\
+#t~ > W- b~ , W- > l- vl~)
+#add process p p > t~ t~ j / s0 QCD= 99 , (t~ > W- b~ , W- > l- vl~), (\
+#t~ > W- b~ , W- > l- vl~)
+#add process p p > t t j j / s0 QCD= 99 , (t > W+ b , W+ > l+ vl),    (\
+#t > W+ b , W+ > l+ vl)
+#add process p p > t~ t~ j j / s0 QCD= 99 , (t~ > W- b~ , W- > l- vl~),\
+# (t~ > W- b~ , W- > l- vl~)
```
[the remark this answers is not in the document]
Comment (efeyazgan) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/g2HDM/ttt/g2HDM_ttt_a0_proc_card.dat]: done

[the remark this answers is not in the document]
Comment (efeyazgan) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/g2HDM/ttt/g2HDM_ttt_a0_proc_card.dat]: along with a few other fixes

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/g2HDM/ttt/g2HDM_ttt_a0_run_card.dat (hunk 1 above)
Comment (efeyazgan) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/g2HDM/ttt/g2HDM_ttt_a0_run_card.dat:54]: I haven't looked at them for this case. I'll do it later.

**Comment by efeyazgan** [comment]
@Saptaparna @menglu21 here are the DJR plots:
DJR

**Comment by menglu21** [comment]
Hi @efeyazgan are you using qcut=20 for the djr plot, the DJR(1->0) seems not good to me, I think you need to use larger qcut, e.g., 45, if you want to keep xqcut=30

**Comment by efeyazgan** [comment]
Hi @menglu21 Fixing njetmax, and also using qcut 45 GeV, they look much better. 

Canvas

**Pull request merged**